.github/workflows/pull-request.yaml

```diff
@@ -29,3 +29,26 @@ jobs:
         env:
           # We make a rinkeby request in one unit test
           INFURA_KEY: ${{ secrets.INFURA_KEY }}
+  tenderly-ci:
```

Comment:
not needed now anymore, but is there a reason to not have "github CI" a separate PR?

Reply:
I would say the reason was that there were no tests before so what would ci have even been doing?